src/sentry/tasks/summaries/daily_summary.py

```diff
@@ -237,12 +237,15 @@ def build_summary_data(
                 "release_id", flat=True
             )
             releases = Release.objects.filter(id__in=release_projects, date_added__gte=ctx.end)
-            for release in releases[:2]:  # or whatever we limit this to
-                new_groups_in_release = Group.objects.filter(project=project, first_release=release)
-                if new_groups_in_release:
-                    project_ctx.new_in_release = {
-                        release.id: [group for group in new_groups_in_release]
-                    }
+            for release in releases:
+                if len(project_ctx.new_in_release) < 2:  # or whatever we limit this to
```

Comment:
no, when the task runs it gets releases in the last 24 hours that have issues that were first seen in that release

Reply:
Hmm for the loop, it seems like we're overriding `project_ctx.new_in_release` for each release?
If release 1+2 both had 1 issue, `project_ctx.new_in_release` would only have the 1 issue from the 2nd release.

Reply:
I'm curious is it better to limit by the number of first issues we find, instead of the number of releases that have first issues?

Reply:
We have to limit the number of releases just because of message limits, but are you suggesting we prioritize releases with the most issues? I like that idea

Reply:
Err that does sound like a good idea lol. I guess I'm a little confused how this data is showing up on the daily summary as I don't see a section for releases/first issues

Reply:
It isn't showing up cause it's broken 🙃 lol this PR was meant to fix it so it would show up. There's a screenshot in [here](https://github.com/getsentry/sentry/pull/66044) and the code is [here](https://github.com/getsentry/sentry/blob/master/src/sentry/integrations/slack/message_builder/notifications/daily_summary.py#L125-L139)

Reply:
Hmm so we're limiting it to two releases. What if release 1 has like 20 new issues, and release 2 only has 1.

Reply:
Also should the releases be in order of newest to oldest?  I'm not sure what order [this query](https://github.com/getsentry/sentry/pull/67212/files#diff-898cefdef894ea6a47ea5aecbc319518c50e57bb8dba8b1c370d39a74177aabdR239) is showing them

Reply:
The message builder code limits the groups to 3 per release (though I could put that limit earlier). I don't think the order matters so much since it's only releases in the last 24 hours, but rather more problematic releases should take priority. I'm not sure how to quantify that though - if there's a release with 100 new issues with 1 event each, is that higher priority than a release with 1 new issue with 100 events? 🤔

Reply:
Ok just had a meeting about this project and I brought it up - for now we're going to prioritize just getting it to show up at all, and worry about this release/issue prioritization later (or we might rethink how to display this data).